### PR TITLE
Fix Windows build and CI

### PR DESCRIPTION
**Pull request by nmellado**
Update third party repositories
Fix some compilation errors 
Add continuous integration

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    .travis.yml
+    .gitignore
+
+# specify custom environment variables
+environment:
+  MSVC_DEFAULT_OPTIONS: ON
+  RA_REPOSITORY_ROOT: C:\projects\Radium-Engine
+  RA_BUILD_DIR: C:\projects\Radium-Engine\build
+  RA_BUNDLE_DIR: C:\projects\Radium-Engine\Bundle-MSVC
+  RA_RELEASE_NAME: Radium-Engine-%platform%-%configuration%
```
Comment (louen) [inline appveyor.yml]: These absolute paths are going to be a problem for people who use this location

Comment (louen) [reply appveyor.yml]: Oh wait. Is it the CI script ?

Comment (nmellado) [reply appveyor.yml]: It's only for AppVeyor VMs, what's the problem ?
You are not supposed to use this script on your computer.

Comment (louen) [reply appveyor.yml]: OK. my bad.

**Comment by nmellado** [comment]
This PR fixes #270 .

**Comment by nmellado** [comment]
PR ready for review.

**Comment by hoshiryu** [comment]
I tried to build and run on windows: all went fine, except for the picking shaders which are not compiling, thus preventing picking to be performed (app crashes)

**Comment by nmellado** [comment]
What kind of error do you have ?
Do you think it could be due to your drivers/GPU configuration ?

It’s hard to test shaders compilation with CI.

**Comment by hoshiryu** [comment]
`
#critical: Linker error:
Type mismatch: Type of out_eltID different between shaders.
Out of resource error.

- 20:24:22 ERROR: Error occurred while loading shader program PickingPoints :
Default shader program used instead.

#critical: Linker error:
Type mismatch: Type of out_eltID different between shaders.
Out of resource error.

- 20:24:22 ERROR: Error occurred while loading shader program PickingLines :
Default shader program used instead.

#critical: Linker error:
Type mismatch: Type of out_eltID different between shaders.
Out of resource error.

- 20:24:22 ERROR: Error occurred while loading shader program PickingTriangles :
Default shader program used instead.
`

This was due to a missing "flat" when declaring an output that is "flat" input on next stage.
But I still can't pick, there might still be something wrong with the shaders (by the way, it must be related with the issue on Mac - PR #281 )

**Comment by nmellado** [comment]
CI Integration System is now ready.
Tests fail on Mac: this should be fixed by PR #276 .

**Pull request merged**